packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -106,73 +95,37 @@ All done! To run your application:
     printStatus(message);
     return 0;
   }
-}
 
-abstract class Template {
-  final String name;
-  final String description;
-  final Map<String, String> files = <String, String>{};
-  final Map<String, dynamic> additionalTemplateVariables = <String, dynamic>{};
-
-  Template(this.name, this.description);
-
-  void generateInto({
-    Directory dir,
-    String flutterPackagesDirectory
-  }) {
-    String dirPath = path.normalize(dir.absolute.path);
+  void _renderTemplates(Directory projectDir, String flutterPackagesDirectory, bool renderDriverTest) {
```

Comment:
Perhaps use a named arg for the ending bool: `{ bool generateDriverTest: false }`